models/public/mixnet-l/mixnet-l.md

```diff
@@ -0,0 +1,77 @@
+# mixnet-l
+
+## Use Case and High-Level Description
+MixNets are a family of mobile-sizes image classification models equipped with MixConv,
+a new type of mixed depthwise convolutions. There are three MixNet architectures -
+`MixNet-S` (Small), `MixNet-M` (Middle), `MixNet-L` (Large). The main differences are using
+MixConv with different kernel sizes and number of layers. Using `MixNet-L` allows to achieve greater accuracy.
+Use this [link](https://arxiv.org/abs/1907.09595) to learn more about MixNet architectures.
+`MixNet-L` was pretrained in TensorFlow\*.
+All the MixNet models have been pretrained on the ImageNet\* image database.
+For details about this family of models, check out the [TensorFlow Cloud TPU repository](https://github.com/tensorflow/tpu/tree/master/models/official/mnasnet/mixnet).
+
+## Example
```

Comment:
we are removing empty section from model documentation (see [PR-1749](https://github.com/openvinotoolkit/open_model_zoo/pull/1749)), please provide sample image or remove empty section